diff --git a/Makefile b/Makefile
index 9d48bff..7516f35 100644
--- a/Makefile
+++ b/Makefile
@@ -35,7 +35,7 @@ BIN=bin
 
 LIB=$(shell pwd)/urb
 
-RM=rm -f
+RM=rm -rf
 ifneq ($(UNAME),FreeBSD)
 CC=gcc
 CXX=g++
diff --git a/include/noun/allocate.h b/include/noun/allocate.h
index 4ad1e4c..451eef5 100644
--- a/include/noun/allocate.h
+++ b/include/noun/allocate.h
@@ -22,7 +22,7 @@
   **/
     /* u3a_bits: number of bits in word-addressed pointer.  29 == 2GB.
     */
-#     define u3a_bits  29
+#     define u3a_bits  28
 
     /* u3a_page: number of bits in word-addressed page.  12 == 16Kbyte page.
     */
diff --git a/outside/commonmark/man/man3/cmark.3 b/outside/commonmark/man/man3/cmark.3
index e5b55fd..3eeb928 100644
--- a/outside/commonmark/man/man3/cmark.3
+++ b/outside/commonmark/man/man3/cmark.3
@@ -1,4 +1,4 @@
-.TH cmark 3 "December 05, 2014" "LOCAL" "Library Functions Manual"
+.TH cmark 3 "November 07, 2015" "LOCAL" "Library Functions Manual"
 .SH NAME
 
 .B cmark
