diff --git a/Makefile b/Makefile
index a80d7bf..b35b79a 100644
--- a/Makefile
+++ b/Makefile
@@ -35,7 +35,7 @@ BIN=bin
 
 LIB=$(shell pwd)/urb
 
-RM=rm -f
+RM=rm -rf
 ifneq ($(UNAME),FreeBSD)
 CC=gcc
 CXX=g++
diff --git a/include/noun/allocate.h b/include/noun/allocate.h
index 4ad1e4c..451eef5 100644
--- a/include/noun/allocate.h
+++ b/include/noun/allocate.h
@@ -22,7 +22,7 @@
   **/
     /* u3a_bits: number of bits in word-addressed pointer.  29 == 2GB.
     */
-#     define u3a_bits  29
+#     define u3a_bits  28
 
     /* u3a_page: number of bits in word-addressed page.  12 == 16Kbyte page.
     */
