diff --git a/Makefile b/Makefile
index 62b28de..6a60573 100644
--- a/Makefile
+++ b/Makefile
@@ -58,7 +58,7 @@ INCLUDE=include
 MDEFINES=-DU2_OS_$(OS) -DU2_OS_ENDIAN_$(ENDIAN) -D U2_LIB=\"$(LIB)\"
 
 # NOTFORCHECKIN - restore -O2
-CFLAGS= -O2 -g -msse3 -ffast-math \
+CFLAGS= -O2 -g -ffast-math \
 	-funsigned-char \
 	-I/usr/local/include \
 	-I/opt/local/include \
diff --git a/v/main.c b/v/main.c
index 3260641..4398e42 100644
--- a/v/main.c
+++ b/v/main.c
@@ -18,8 +18,6 @@
 #include <termios.h>
 #include <term.h>
 #include <dirent.h>
-#include <pmmintrin.h>
-#include <xmmintrin.h>
 
 #define U2_GLOBAL
 #define C3_GLOBAL
@@ -255,9 +253,6 @@ main(c3_i   argc,
 {
   c3_w kno_w;
 
-  _MM_SET_FLUSH_ZERO_MODE(_MM_FLUSH_ZERO_ON);
-  _MM_SET_DENORMALS_ZERO_MODE(_MM_DENORMALS_ZERO_ON);
-
   //  Parse options.
   //
   if ( u2_no == _main_getopt(argc, argv) ) {
